components/04-views/dashboard/returns/returns.hbs

```diff
@@ -0,0 +1,17 @@
+{{ render '@header' }}
+
+<div class="dashboard container">
+    <div class="grid">
+        <div class="grid__column grid__column--1-4 dashboard__sidebar">
+            {{ render '@dashboard-nav' }}
+        </div>
+
+        <div class="grid__column grid__column--3-4 dashboard__content">
+            {{> '@dashboard-table' returns }}
+
+            {{> '@dashboard-actions' buttons }}
+        </div>
+    </div>
+</div>
+
+{{ render '@footer' }}
```

Comment:
Returns default doesn't seem to display well... check the view please

Reply:
fixed

Reply:
👍 
100% button for mobile.

Reply:
changed